src/Container/ApplicationFactory.php

```diff
@@ -77,19 +63,13 @@ public function __invoke(ContainerInterface $container)
             ? $container->get(RouterInterface::class)
             : new FastRouteRouter();
 
-        $finalHandler = ! empty($config['zend-expressive']['raise_throwables'])
-            ? $this->marshalNoopFinalHandler($container)
-            : $this->marshalLegacyFinalHandler($container, $config);
+        $delegate = $this->marshalNoopFinalHandler($container);
```

Comment:
We still need a "final handler" of some sort for cases where the internal queue is exhausted, which is why I marshal the `NoopFinalHandler` here. What needs to happen is documenting the use case; it should likely either return a 404, a 400, or a 500 response, and how it does so will vary from application to application. Making a not of that now.